assets/src/scss/blocks/core-overrides/BlockSpacing.scss

```diff
@@ -36,7 +36,6 @@
   }
 }
 
-.wp-block-quote,
```

Comment:
With this condition, the Quote would stick to the left in the editor in larger screens